### PR TITLE
UBER-535

### PR DESCRIPTION
**Pull request by BykhovDenis**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: models/calendar/src/migration.ts [hunk 1]
```diff
@@ -13,34 +13,70 @@
...
+  for (const user of users) {
+    if (!existCalendars.has(`${user._id}_calendar` as Ref<Calendar>)) {
+      await tx.createDoc(
+        calendar.class.Calendar,
+        core.space.Space,
+        {
+          name: user.email,
+          description: '',
+          archived: false,
+          private: false,
```
Comment (haiodo) [inline models/calendar/src/migration.ts:35]: Should it be private?

File: packages/ui/src/components/calendar/WeekCalendar.svelte [hunk 2]
```diff
@@ -36,7 +36,7 @@
     : new Date(new Date(currentDate).setHours(0, 0, 0, 0))
 </script>
 
-<table class="antiTable">
+<table class="">
```
Comment (haiodo) [inline packages/ui/src/components/calendar/WeekCalendar.svelte]:
```suggestion
<table>
```

Comment (haiodo) [reply packages/ui/src/components/calendar/WeekCalendar.svelte]: Why use table without class?

File: plugins/calendar-resources/src/components/IntegrationConnect.svelte [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
```
Comment (haiodo) [inline plugins/calendar-resources/src/components/IntegrationConnect.svelte]:
```suggestion
// Copyright © 2023 Hardcore Engineering Inc.
```

File: plugins/calendar-resources/src/components/icons/Calendar.svelte [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
```
Comment (haiodo) [inline plugins/calendar-resources/src/components/icons/Calendar.svelte]:
```suggestion
// Copyright © 2023 Hardcore Engineering Inc.
```

File: plugins/calendar-resources/src/components/icons/Calendar.svelte [hunk 5]
```diff
@@ -0,0 +1,68 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
```
Comment (haiodo) [inline plugins/calendar-resources/src/components/icons/Calendar.svelte]:
```suggestion
```

**Review by haiodo — APPROVED** [review]

File: models/calendar/src/migration.ts (hunk 1 above)
Comment (BykhovDenis) [reply models/calendar/src/migration.ts:35]: No, we should have possibility to show events to another users in future

File: plugins/calendar-resources/src/components/IntegrationConnect.svelte (hunk 3 above)
Comment (BykhovDenis) [reply plugins/calendar-resources/src/components/IntegrationConnect.svelte]: Fixed

File: plugins/calendar-resources/src/components/icons/Calendar.svelte (hunk 4 above)
Comment (BykhovDenis) [reply plugins/calendar-resources/src/components/icons/Calendar.svelte]: Fixed

File: plugins/calendar-resources/src/components/icons/Calendar.svelte (hunk 5 above)
Comment (BykhovDenis) [reply plugins/calendar-resources/src/components/icons/Calendar.svelte]: Fixed

**Review by haiodo — APPROVED** [review]

**Pull request merged**